Wrapping/WrapITKTypes.cmake

```diff
@@ -148,6 +148,13 @@ WRAP_TYPE("itk::FixedArray" "FA" "itkFixedArray.h")
     ADD_TEMPLATE("${ITKM_SC}${d}" "${ITKT_SC},${d}")
     ADD_TEMPLATE("${ITKM_B}${d}"  "${ITKT_B},${d}")
   endforeach()
+
+  # Wrap FixedArray for BSplineInterpolationWeightFunction:
+  ADD_TEMPLATE("${ITKM_D}9" "${ITKT_D},9")
+  ADD_TEMPLATE("${ITKM_D}16" "${ITKT_D},16")
+  ADD_TEMPLATE("${ITKM_UL}16" "${ITKT_UL},16")
```

Comment:
Shouldn't D and F be enough? Can F be chosen for interpolation precision? What sense does it make to wrap UL?

Reply:
> Shouldn't D and F be enough? Can F be chosen for interpolation precision? What sense does it make to wrap UL?

@dzenanz Honestly I'm clueless. https://open.cdash.org/viewBuildError.php?type=1&buildid=7441714 says:

> itkBSplineBaseTransform: warning(4): ITK type not wrapped, or currently not known: itk::FixedArray< unsigned long, 16 >

Just doing trial and error now, but please tell me if you have a suggestion!

Reply:
If the build system asks for it, add it 😄

That answers all my previous questions.